src/cli.rs

```diff
@@ -9,7 +10,29 @@ pub mod preprocessors;
 pub mod utils;
 
 pub fn cli() -> Command {
-    command!().arg_required_else_help(true).codecs()
+    command!()
+        .arg_required_else_help(true)
+        .after_help(indoc! {r#"List of supported codecs
```

Comment:
I would like to know if Rust supports displaying the syntax in Markdown format directly, or we should re-format it to make the display of the command-line interface clear and easy to understand.

Reply:
We should format it by hand